src/components/PersonLink.tsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import { Person } from '../types';
+import { Link } from 'react-router-dom';
+import classNames from 'classnames';
+
+type Props = {
+  person: Person | undefined;
+  personName: string;
+};
```

Comment:
```suggestion
type Props = {
  person: Person;
};
```
- person is required, it doesn't make sense to render component without it
- name you can get from the person object